### PR TITLE
CSS logger: omit inlined ad styles

### PR DESCRIPTION
**Pull request by stephanfowler**

### CODE REVIEW

**Comment by robertberry** [comment]
Should probably be `js-gu-loggable`

File: common/app/views/fragments/javaScriptLaterSteps.scala.html [hunk 1]
```diff
@@ -129,6 +129,7 @@
                     head = document.getElementsByTagName('head')[0];
                     s.innerHTML = c;
                     s.setAttribute('data-loaded-from', 'local');
+                    s.className = "gu-loggable";
```
Comment (phamann) [inline common/app/views/fragments/javaScriptLaterSteps.scala.html]: Will most likely have a merge conflict with #8205 so make sure to get @sndrs to add it to the new link elements.

**Comment by phamann** [comment]
:+1: bar comments.

File: common/app/views/fragments/javaScriptLaterSteps.scala.html (hunk 1 above)
Comment (stephanfowler) [inline common/app/views/fragments/javaScriptLaterSteps.scala.html]: Yep good call. (It's only needed on `style` elements though.)

**Pull request merged**